components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Services/DefaultSessionManager.cs

```diff
@@ -163,7 +163,7 @@ public class DefaultSessionManager : ISessionManager, IDisposable {
                     return;
                 }
                 if (wrapper._subscriptions.TryRemove(subscription.Id, out _)) {
-                    _logger.Information("Subscription '{subscriptionId}' unregistered from session '{sessionId}' in state {state}",
+                    _logger.Verbose("Subscription {subscriptionId} unregistered from session {sessionId} in state {state}",
```

Comment:
>Verbose [](start = 28, length = 7)

this is typically is happening maximum once during runtime if the cleanup works properly, I would recommend to keep it at least at debug level

Reply:
2825 hits in 10 min, do you insist?